search/serializers.py

```diff
@@ -162,14 +171,16 @@ def _serialize_comment_tree_for_bulk(comments):
     Serialize an iterable of Comment and their replies for a bulk API request
 
     Args:
-        comments (iterable of Comment): An iterable of comments
+        comments (iterable of str): An iterable of comment ids
```

Comment:
Can we change `comments` to `comment_ids`?

Reply:
✔️